service/src/main/resources/application.yml

```diff
@@ -90,6 +90,9 @@ profile:
     client-credential-file-path: build/resources/main/generated/bpm-client-sa.json
     base-path: ${env.tps.basePath}
 
+  gcp:
+    sa-credential-file-path: ${GOOGLE_APPLICATION_CREDENTIALS:build/resources/main/generated/bpm-client-sa.json}
```

Comment:
Reference https://github.com/broadinstitute/terra-helmfile/blob/c769c4763340af04f5c68356be2fb195e0f6c1a3/charts/bpm/templates/deployment.yaml#L294 when  not running locally.